hummingbird/ml/operator_converters/sklearn/missing_indicator.py

```diff
@@ -0,0 +1,57 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Converter for scikit-learn MissingIndicator.
+"""
+from .._base_operator import BaseOperator
+import numpy as np
+from onnxconverter_common.registration import register_converter
+import torch
+
+
+class MissingIndicator(BaseOperator, torch.nn.Module):
+    """
+    Class implementing Imputer operators in MissingIndicator.
+    """
+
+    def __init__(self, sklearn_missing_indicator, device):
+        super(MissingIndicator, self).__init__()
+        self.transformer = True
+        self.missing_values = torch.nn.Parameter(
+            torch.tensor([sklearn_missing_indicator.missing_values], dtype=torch.float32), requires_grad=False
+        )
+        self.features = sklearn_missing_indicator.features
+        self.is_nan = True if (self.missing_values in ["NaN", None, np.nan]) else False
```

Comment:
@ksaur Can you try changing this to
`self.is_nan = True if (sklearn_missing_indicator.missing_values in ["NaN", None, np.nan]) else False`

Reply:
Ah!! Great catch!!! THANKS.

(You must be some kind of wizard think.)